aloha-h2o/src/main/scala/com/eharmony/aloha/models/h2o/json/H2oModelJson.scala

```diff
@@ -89,6 +98,21 @@ case class DoubleH2oSpec(name: String, spec: String, defVal: Option[Double]) ext
   def refInfo = RefInfo[Double]
 }
 
+case class DoubleSeqH2oSpec(name: String, spec: String, size: Int, defVal: Option[Seq[Double]]) extends H2oSpec {
+  type A = Seq[Double]
+  def ffConverter[B] = f => DoubleSeqFeatureFunction(f, size)
+  def refInfo = RefInfo[Seq[Double]]
+
+  protected def sizeErr: String = s"feature '$name' output size != $size"
```

Comment:
Can we include the actual size in the error message.

Reply:
Done.  This was a pain in the butt.  But looks nice now.